examples/twitter/tweet_at_cozmo.py

```diff
@@ -182,20 +182,22 @@ def get_command(self, command_name):
 
     def extract_command_from_string(self, in_string):
         '''Separate inString at each space, loop through until we find a command, return tuple of cmd_func and cmd_args'''
+        first_split_string = in_string.split(';')
```

Comment:
i think the split_string naming becomes a bit confusing now there are two - I think e.g. `command_strings = in_string.split(';')` would make it clearer that the first split is into sub-commands, and "split_string" could then also be renamed into command_args